models/User.php

```diff
@@ -67,6 +76,26 @@ public function rules()
             ['status', 'default', 'value' => self::STATUS_ACTIVE],
             ['status', 'in', 'range' => [self::STATUS_ACTIVE, self::STATUS_DELETED]],
             ['status', 'filter', 'filter' => 'intval'],
+
+            // Username
+            [['username', 'fullname'], 'string'],
+
+            // Avatar
+            ['avatar', 'string', 'max' => 60],
+            ['avatarFile', 'file', 'extensions' => 'png, jpg'],
+        ];
+    }
+
+    /**
+     * @return array
+     */
+    public function scenarios()
+    {
+        static $defaultAttributes = ['username', 'fullname', 'avatar', 'status'];
```

Comment:
Status should not be in default attributes, only in manage.

Reply:
Avatar isn't set directly so I think it's a bad idea to list it here.

Reply:
fix it